src/Command/CommandGenerator.php

```diff
@@ -48,9 +48,11 @@ public function sshCommand(string $destination, string $currentHost, string $com
      * @param string $to
      * @param string $currentHost
      * @param bool $all
+     * @param bool $noFiles
+     * @param bool $noDatabases
      * @return string[]
      */
-    public function syncCommands(string $from, string $to, string $currentHost, bool $all): array
+    public function syncCommands(string $from, string $to, string $currentHost, bool $all, bool $noFiles, bool $noDatabases): array
```

Comment:
the function calls are getting way to big now... we should maybe consider an option dto